<launch>
  <!-- Including the base launch file -->
  <include file="$(find mercator_sensor_fusion_ros_pkg)/launch/base_launch_files/ground_truth_publisher.launch">
    <!-- You can pass arguments here if needed, or override them -->
    <arg name="robot_names" value="['base_link_23', 'base_link_22']"/>
    <arg name="reference_robot_name" value="base_link_23"/>
    <arg name="reference_robot_rotation" value="90.0"/>
    <arg name="new_frame_id" value="robot"/>
    <arg name="tf" value="/tf"/>
    <arg name="ground_truth_poses" value="/ground_truth_poses"/>
  </include>

  <include file="$(find mercator_sensor_fusion_ros_pkg)/launch/base_launch_files/calibration_cam_lidar.launch">
    <!-- Optionally override arguments here -->
    <arg name="collect_time" value="60.0"/>
    <arg name="data_max_size" value="10000"/>
    <arg name="cam_poses" value="/cam_poses"/>
    <arg name="lidar_poses" value="/lidar_poses"/>
    <arg name="ground_truth_poses" value="/ground_truth_poses"/>
  </include>
</launch>